### ISSUE TITLE
Light miner: Diagnostics page should NOT display blockchain details

### ISSUE THREAD

**Issue by marvinmarnold** [NebraLtd/hm-diag#321]
Miners no longer keep track of block height.

**Acceptance criteria**
- Miner container must be using hm-gatewayrs
- `Sync Percentage` removed
- `Miner Connected To Blockchain` removed
- `Height Status` renamed to `Validator height`
- Add `Validator block age`
- Add `Validator address`
- `Miner Relayed` removed from diagnostics page
- upnp removed from hm-config

**Comment by shawaj** [comment]
Maybe this should be conditional initially? As we want to keep this until the full move

**Comment by marvinmarnold** [comment]
@shawaj the [plan](https://github.com/NebraLtd/helium-miner-software/issues/399), we decided not to use feature flags and to instead work on a branch and plan to merge into production in discrete step.

**Comment by marvinmarnold** [comment]
@pritamghanghas can you move this issue to the correct swim lane?

**Comment by pritamghanghas** [comment]
Pausing work on this to work on light-hotspot QC [ticket](https://github.com/NebraLtd/hnt-products/issues/7). Will try to push the work in parallel if I find some time. 

Some potential issues for light-hotspot that will need some thought and work:
- We are using [rasp01.deb](https://github.com/NebraLtd/hm-gatewayrs/blob/main/Dockerfile#L13) from gateway-rs releases. This will work only on raspberry pi but not on rock pi. On Rockpi boards, the binary just hangs up. created a new [issue](https://github.com/NebraLtd/hm-gatewayrs/issues/30) for it.
- we are pulling gateway-mfr binary and putting it in the pypi package as a github action. I think it makes more sense to checkin this binary into our hm-pyhelper repo. In light hotspot we install from a git branch as we have not started publishing the light-spot branch and that leads to this binary being missing.

**Comment by shawaj** [comment]
RasPi 1 Deb should work on RockPi too. Don't see why it wouldn't. Are you sure that's the reason the container is hanging?

**Comment by pritamghanghas** [comment]
> RasPi 1 Deb should work on RockPi too. Don't see why it wouldn't. Are you sure that's the reason the container is hanging?
My bad on this one. While raising the issue, I confused it with gateway-mfr-rs. My issue was with gateway-mfr-rs. I will have to reconfirm which one was pulled for that.
